sdk/core/core-autorest/package.json

```
@@ -0,0 +1,122 @@
+{
+  "name": "@azure/core-autorest",
+  "version": "1.0.0-preview.1",
+  "private": true,
+  "description": "Core library for interfacing with AutoRest generated code",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "types/latest/core-autorest.d.ts",
+  "typesVersions": {
+    "<3.6": {
+      "types/latest/src/*": [
+        "types/3.1/src/*"
+      ]
```

I don't think `src` will appear it `types/latest` or `types/3.1` since we rollup the types into a bundle.
```suggestion
    "<3.6": {
      "types/latest/*": [
        "types/3.1/*"
      ]
```

---

it does show up, but are you saying we shouldn't include the raw .d.ts and only the rolled up file?

---

Right. I don't think there's a practical way for a user to actually use the raw .d.ts files.